mailing-list-api/server.js

```diff
@@ -0,0 +1,60 @@
+import express from "express";
+
+const app = express();
+app.use(express.json());
+
+const lists = new Map();
+lists.set("staff", ["talea@techtonica.org", "michelle@techtonica.org"]);
+lists.set("cohort-h1-2020", ["ali@techtonica.org", "humail@techtonica.org", "khadar@techtonica.org"]);
```

Comment:
THis is an intresting approach - is there any particulaur reason you moved the data out of the file?